lmfdb/groups/abstract/templates/abstract-show-subgroup.html

```diff
@@ -6,7 +6,7 @@
 <h2> Subgroup ($H$) information</h2>
 <p>
   <table>
-    <tr><td><a href="{{url_for('.by_label', label=seq.subgroup)}}">${{seq.subgroup_tex}}$</a></td><td>  </td></tr>
+    <tr><td>{{KNOWL('group.name', title='Description:')}}</td><td><a href="{{url_for('.by_label', label=seq.subgroup)}}">${{seq.subgroup_tex}}$</a></td></tr>
```

Comment:
We should also add the knowl for the other places names show up (the ambient group and quotient group on the subgroup page, for example).